files/zimbra_config.py

```diff
@@ -64,7 +70,25 @@ def create_config():
     if not os.path.isdir("/srv/zimbraweb/mnt/logs"):
         os.mkdir("/srv/zimbraweb/mnt/logs")
         os.chmod("/srv/zimbraweb/mnt/logs", 0o777)
-    
+
+def create_config_env():
+    config = {
+        "zimbra_host": "https://studgate.dhbw-mannheim.de",
+        "email_domain": "student.dhbw-mannheim.de",
+        "smtp_fallback": "disabled",
+        "smtp_fallback_relay_host": "172.17.0.2",
+    }
+
+    for key, default in config.items():
+        config[key] = os.getenv(key, default)
+
+    with open(CONF_PATH, "w") as f:
+        json.dump(config, f, indent=4)
+
+    if not os.path.isdir("/srv/zimbraweb/mnt/logs"):
+        os.mkdir("/srv/zimbraweb/mnt/logs")
+        os.chmod("/srv/zimbraweb/mnt/logs", 0o777)
+
 def get_config() -> Dict[str, str]:
     validate_config()
     with open(CONF_PATH, "r") as f:
```

Comment:
This was added by me but just letting you know I want to fix this with this PR. Now that I'm looking at the code this doesn't make any sense.  It calls `validate_config` but then does nothing if this call returns False. 

How should we handle an invalid option? Fallback to default? Or throw an error and exit?

Reply:
I think `validate_config()` should look if the file is valid json and if missing options fall back to default. Throw error only if the json is not valid. Write to log if option was missing.